db/knex_migrations/2024-03-24-0000-add-datadog-monitor-type.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex) {
+    // Add new columns monitor.datadog*
+    return knex.schema
+        .alterTable("monitor", function (table) {
+            table.string("datadog_site", 255);
+            table.string("datadog_api_key", 255);
+            table.string("datadog_app_key", 255);
```

Comment:
Is one of these a password? If yes this could use the existing Rename `radiusPassword` (has to be renamed to monitor.password for general use) field